### PR TITLE
sender should have the option to decide

### LINKED ISSUE #89: Vittorio: Vagueness around consent
“I do not have specific text suggestions, but I noted some kind of vagueness around consent, i.e. in the subjects of sentences like "The sender needs to agree to sending the information" - who is the sender, the application or the user? One thing is the agreement of the client application that will share information, i.e. of its developer, and another is the agreement of the user that will be sitting in front of that application and entering their information into it.
 
There is a paragraph in 2.2 discussing this, but it only mentions the case in which the application would like to collect information "against" the user, not other cases which may appear, such as legal compliance. Also, it makes some blanket statements that are really too broad, such as "Information that directly relates to the user should not be shared without the user's consent" - there are a cases in which user consent is not legally necessary nor even possible, but applications will be required to share that information anyway. I would just refer to applicable privacy legislation, declaring this out of scope.”

It is unclear to me what we should do about this. But I think we were vague to some extent on purpose here.

See https://mailarchive.ietf.org/arch/msg/architecture-discuss/k4npcAf_EcbpP1Dp8OY1cDwFE8c/

### PR DESCRIPTION
**Pull request by mirjak**
fixes #89

Is that sufficient?

### CODE REVIEW

File: draft-iab-path-signals-collaboration.md [hunk 1]
```diff
@@ -322,14 +322,15 @@ the distribution of information, in order to give adequate control to
 each entity over the collaboration or information sharing. This can
 be achieved as discussed below.
 
-The sender needs to agree to sending the information.
+The sender should have an option to decide that it wants to send the information
+to a specific entities or set of entities.
```
Comment (tfpauly) [inline draft-iab-path-signals-collaboration.md:326]:
```suggestion
The sender needs to decide that it is willing to send information to a specific entity or
set of entities.
```

File: draft-iab-path-signals-collaboration.md [hunk 2]
```diff
@@ -322,14 +322,15 @@ the distribution of information, in order to give adequate control to
...
 Any passing of information or request for an action needs to be explicit,
 and use signalling mechanisms that are designed for the purpose.
 Merely sending a particular kind of packet to a destination should not
 be interpreted as an implicit agreement.
 
 At the same time, the recipient of information or the target of a
-request should agree to receiving the information. It
+request should have the option to agree or deny to receiving the information. It
```
Comment (tfpauly) [inline draft-iab-path-signals-collaboration.md:333]:
```suggestion
request needs to decide that it is willing to receive the information. It
```

**Review by tfpauly — CHANGES_REQUESTED** [review]

**Comment by mirjak** [comment]
@tfpauly your wording is find but that doesn't capture the aspect that the protocol design actually needs to have a option to configure. I guess that was too subtle in my sentence as well. Should we add a separate sentence maybe?

**Comment by mirjak** [comment]
Yes, this beyond is beyond what's raised in the issue. It's not about the application is doing but what to consider in protocol design. I was think about quic and the spin bit and basically saying that there should be an option in the proposal to disable it.

**Comment by tfpauly** [comment]
Ah, I see — a protocol/spec-level option, not a runtime or software configuration option. That's not super clear...

**Comment by mirjak** [comment]
Isn't the whole document about protocol design recommendations...? ;-)

How can we make that more clear?

**Review by jariarkko — APPROVED** [review]
I think we are good to go. I will apply this PR later today if I don't hear otherwise.

**Review by tfpauly — APPROVED** [review]

**Review by jariarkko — APPROVED** [review]

**Comment by jariarkko** [comment]
Based on chat discussion I'm applying this as originally suggested.

**Pull request merged**